packages/rpc/src/binance/models/BinanceNodeInfo.ts

```diff
@@ -0,0 +1,45 @@
+import 'reflect-metadata';
+import { Type, Transform } from 'class-transformer';
+import BigNumber from 'bignumber.js';
+ 
+class NodeInfo {
+    network: String
```

Comment:
`string` should be lowercased. Please review all string type defintion in the code.